controllers/provisioners/eks/scaling/interface.go

```diff
@@ -57,4 +57,12 @@ type CreateConfigurationInput struct {
 	Volumes               []v1alpha1.NodeVolume
 	UserData              string
 	SpotPrice             string
+	LicenseSpecifications []string
+	Placement             *LaunchTemplatePlacementInput
+}
+
+type LaunchTemplatePlacementInput struct {
```

Comment:
Do we need to have a private model *and* a public model? I notice we just have `v1alpha1.NodeVolume` and no internal representation of a volume.